discovery_engine_core/web-api/openapi.yaml

```
@@ -145,6 +145,12 @@ components:
       description: A user id can be any non-empty UTF-8 string that does not contain the null byte.
       type: string
       example: "user id"
+    BaseError:
+      type: object
```

🔧 if the id is not optional
```suggestion
      type: object
      required: [request_id]
```

---

I didn't put it optional because I'm not sure we can always return it. Change it later from optional to required does not break the api.